bindings/python/crocoddyl/multibody/multibody.cpp

```diff
@@ -30,6 +30,9 @@ void exposeMultibody() {
   exposeDataCollectorImpulses();
   exposeDifferentialActionFreeFwdDynamics();
   exposeDifferentialActionContactFwdDynamics();
+#if PINOCCHIO_VERSION_AT_LEAST(2, 9, 0)
```

Comment:
Remove this macro as it is not needed (we have implemented in the expose function).